bin/merge_and_align_libraries_from_easypqp.py

```diff
@@ -130,8 +130,10 @@ def combine_libs_by_edges_of_MST(T, rsq_threshold):
 
         cols=[c for c in reference.columns if not 'NormalizedRetentionTime' in c and not 'TransitionId' in c]
         aligned = align_libs(reference, other, rsq_threshold)
+        seqs_ref=reference['ModifiedPeptideSequence'].values.tolist()
+        aligned = aligned[~aligned['ModifiedPeptideSequence'].isin(seqs_ref)]
         concat = pd.concat([reference,aligned])
-        regrouped = concat.iloc[concat[cols].drop_duplicates().index]
+        regrouped = concat  #.iloc[concat[cols].drop_duplicates().index]
```

Comment:
what about the commented out part?

Reply:
Yeah maybe I should remove the commented parts, initially I kept it in case I wanted to reuse the code at some point.

In the beginning I was merging libraries and then dropping the duplicated entries. Now I keep all spectra from the reference and only add spectra of distinct peptides to the merged library.